src/index.tsx

```diff
@@ -74,10 +74,13 @@ export function hasRequiredDataForTemplate({
 }): boolean {
   const template: Template | undefined = TEMPLATES[templateName];
   if (!template) throw new Error(`Template "${templateName}" not found`);
-  for (const path of template.requirements) {
-    if (!hasValue(passport.data, path)) {
+  for (const {key, value} of template.requirements) {
+    if (!hasValue(passport.data, key)) {
       return false;
     }
+    if (value) {
+      return getString(passport.data, key) === value;
+    }
```

Comment:
:+1: simple & flexible! allowing template requirements to be key=value or just existance-of-key checks makes much more sense to me when considering how templates link to existing service content